### PR TITLE
Feature: preview opt out

### LINKED ISSUE #165: Add a Preview button on Graph Explorer classic to switch and remember to use v2
Portal office admin center has this feature like https://admin.microsoft.com/Adminportal/Home?source=applauncher#/homepage

image
and they can get back if not happy
image

### LINKED ISSUE #213: Move the profile to the sidebar
**Is your feature request related to a problem? Please describe.**
When hosted on the portal, the sign in button becomes confusing as the portal has it's own sign in button in the same location

**Describe the solution you'd like**
Move the sign in button to the sidebar above the sample queries

**Describe alternatives you've considered**
Removing the sign in button from the portal on load

### PR DESCRIPTION
**Pull request by cwahome**
## Overview
Adds a toggle to the sidebar to move from preview to classic
Also moves the authentication component to the sidebar, and fixes it for smaller screens

Fixes #165 
Fixes #213 

### Demo

Logged in desktop
image

Logged in mobile
image

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Run GE Classic note via terminal
* After GE is running, run v2 via terminal
* When both are running go to v2 and switch the toggle. 
* Notice that it reloads the page with Classic instead of v2

### CODE REVIEW

**Comment by jobala** [comment]
@cwahome how do I test this?

**Comment by jobala** [comment]
@cwahome how do I test this?

File: .vscode/settings.json [hunk 1]
```diff
@@ -8,5 +8,28 @@
         "**/node_modules": true,
         "**/.github": true,
     },
-    "files.trimTrailingWhitespace": true
-}
\ No newline at end of file
+    "files.trimTrailingWhitespace": true,
```
[the remark this answers is not in the document]
Comment (cwahome) [reply .vscode/settings.json]: nope

File: src/app/views/authentication/profile/Profile.tsx [hunk 2]
```diff
@@ -130,11 +132,38 @@ export class Profile extends Component<IProfileProps, IProfileState> {
...
+            <Card.Item fill={true}>
+            <Persona {...persona} coinSize={80} size={PersonaSize.size40} hidePersonaDetails={true} />
+            </Card.Item>
+          <Card.Section>
+            <span className={classes.personaText}>
+              {persona.text}
+            </span>
+            <span className={classes.personaSecondaryText}>{persona.secondaryText}</span>
+            <ActionButton ariaLabel='profile' role='button' menuProps={menuProperties}>
+              More actions
```
[the remark this answers is not in the document]
Comment (cwahome) [reply src/app/views/authentication/profile/Profile.tsx]: Yup.. Resolved in #221

**Pull request merged**